test/FxAccountsPairingChannel.js

```diff
@@ -40,7 +40,7 @@ describe('FxAccountsPairingChannel', () => {
 
   it('throws when network connection fails', async () => {
     await assert.throwsAsync(async () => {
-      await PairingChannel.create('wss://this.should.not.work');
+      await PairingChannel.create('wss://nonexistent.example.com');
```

Comment:
This test stopped working on the 25th of March, when some enterprising soul registered the `not.work` domain and made it redirect to some sort of anarchist manifesto. It would be better if we didn't hit the network in these tests at all, but as  stop-gap this changes it to use a domain that's explicitly reserved for example purposes.